class.metrilo-admin-settings.php

```diff
@@ -11,6 +11,7 @@ class Metrilo_Admin_Settings extends WC_Integration
         private $events_queue = [];
         private $has_events_in_cookie = false;
         private $identify_call_data = false;
+        private $user_tags = false;
```

Comment:
This is supposed to be an array, right? Setting it to a boolean value is deceiving. Better set it to an empty array.